idd/Energy+.idd.in

```diff
@@ -70103,7 +70108,7 @@ WaterHeater:HeatPump:PumpedCondenser,
        \note proportional to the schedule value (i.e. 0.4 = 40% outdoor air, 60% zone air).
   A29, \field Tank Element Control Logic
        \type choice
-       \key MutuallyExlcusive
+       \key MutuallyExclusive
```

Comment:
The [code](https://github.com/NREL/EnergyPlus/blob/2dd86cbb4d3da5d3c1fade4d8b478d627005060d/src/EnergyPlus/WaterThermalTanks.cc#L2391) is expecting MutuallyExclusive, so we should change any files that inadvertently used this wrong key choice.